README.md

```diff
@@ -1,5 +1,75 @@
 # Swift — Introduction From Objective-C
 
-## Objectives
+## Introduction
+
+While Swift is comparatively a young language, having only been publicly announced in June of 2014, the foundational work towards adopting it was begun in 2010 when Apple attracted the efforts of [Chris Lattner](https://en.wikipedia.org/wiki/Chris_Lattner), the [creator of LLVM](http://nondot.org/sabre/). Changes to the canonical 30-year-old programming language Objective-C were incorporated to pave the way towards the transition to its replacement. These included the additions of Automatic Reference Counting (ARC) and closures, among other improvements to the Objective-C language. The goal, however, was to create a new language with modern programming capabilities and syntax. 
+
+##### Statically Typed
+
+And indeed, many aspects of developing with Swift are cleaner and quicker. It is a strongly-typed (or "statically-typed") language that attempts to move errors out of the run-time phase and into the compile-time phase; a move with the intent of making deployed code "safer", but with the effect of handling null reference pointers in a different way.
```

Comment:
it's more then just null reference pointers. i'd say with the "effect of requiring the developer to think more defensively before even compiling their code"